### ISSUE TITLE
Much slower runs using deepxde 0.5.0

### ISSUE THREAD

**Issue by tangqi** [lululxvi/deepxde#25]
Hi 
I recently updated the deepxde to 0.5.0 on my macbook. The runs becomes significantly slower. I noticed during the training, the cpu is only limited to around 100%. But it was using 600% previously (using deepxde 0.4.0) I have tensorflow 1.13.

Here is how I install tensorflow and deepxde:
```
conda create --name tf
conda activate tf
conda install -c conda-forge deepxde
```
 Can you take a look at that? Thanks a lot.

-Qi

**Comment by lululxvi** [comment]
You are right. I have noticed this. This is a bug when I add new features. I will fix this these two days.

**Comment by lululxvi** [comment]
Has been fixed in v0.5.1

**Comment by tangqi** [comment]
Can confirm v0.5.1 works well. Thanks a lot for your quick fix!
